src/logging.cpp

```diff
@@ -288,7 +285,9 @@ std::string LogCategoryToStr(BCLog::LogFlags category)
         return "scan";
     case BCLog::LogFlags::ALL:
         return "all";
-    }
+    case BCLog::LogFlags::NONE:
+        Assume(false);
```

Comment:
What's the point of an `Assume(false)` that just falls through to an `assert(false)` ?